src/core/client/admin/routes/Moderate/Queue/Queue.tsx

```diff
@@ -63,8 +60,15 @@ const Queue: FunctionComponent<Props> = ({
   const [conversationModalVisible, setConversationModalVisible] =
     useState(false);
   const [conversationCommentID, setConversationCommentID] = useState("");
+  const [hasModerated, setHasModerated] = useState(false);
```

Comment:
Otherwise we don't know to set selectedComment to 0 after moderating the first comment.